Release 34

```diff
@@ -0,0 +1,24 @@
+# Release -API Management service: April 2023
+A regular Azure API Management service updated was started February 9, 2023. This release concluded roll out March 31, 2023.
+
+### New features, fixes, and improvements
+
+1. We fixed an issue, where the definition names in the API schema would display blank spaces.
+2. We fixed an issue, where the developer portal was hiding default domain, a parameter has been added to include default domain in the hostnames.
+3. We fixed an issue, where the API revisions were not being properly filtered.
+4. We implemented a fix that will allow users to remove built-in groups, previously where able to add built-in groups but could not remove them.
+5. We fixed an issue, where an error was received when creating API schemas with the same namespace.
+6. We fixed an issue, where policies contained all named values which caused a problem for secrets. The policy failed to check if given user had access to secret.
```

Comment:
Can you explain what the first sentence means? I don't understand the "policies contained all named values" part.

Reply:
https://msazure.visualstudio.com/One/_git/141a6ca5-3ad0-4c0f-9992-e52cc9d7a8f9/pullrequest/7319994